turbinia/workers/__init__.py

```diff
@@ -607,6 +611,19 @@ def run_wrapper(self, evidence):
         original_result_id = self.result.id
         evidence.validate()
 
+        # TODO(wyassine): refactor it so the result task does not
+        # have to go through the preprocess stage. At the moment
+        # self.results.setup is required to be called to set it's status.
+        # Check if Task's job is available for the worker.
+        psq_job_manager = list(job_manager.JobsManager.GetJobNames())
```

Comment:
This `psq_job_manager` variable name is kind of confusing here.  Isn't this a list of jobs?  Maybe just `active_jobs`?